src/lumigo_tracer/lumigo_utils.py

```diff
@@ -337,6 +347,15 @@ def report_json(region: Optional[str], msgs: List[dict], should_retry: bool = Tr
     return duration
 
 
+def write_spans_to_file(to_send):
+    file_name = f"{str(hashlib.md5(to_send).hexdigest())}_single"
```

Comment:
```suggestion
    file_name = f"{hashlib.md5(to_send).hexdigest()}_single"
```
Looks like `hexdigest` returns a string:
![image](https://user-images.githubusercontent.com/49228804/122755903-581ea080-d29e-11eb-8e2d-34a5b4a73010.png)